### PR TITLE
Add edit icon and statuses checkbox icons to habit card #2679

### PR DESCRIPTION
**Pull request by idzhersh**
### **Before**
Screenshot (532)
### **After**
Screenshot (534)
Screenshot (535)
Screenshot (537)

### CODE REVIEW

File: src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.html [hunk 1]
```diff
@@ -12,6 +12,7 @@
         <img class="first-photo" [src]="firstFriend" alt="First friend photo">
         <img class="second-photo" [src]="secondFriend" alt="Second friend photo">
       </div>
+      <div class="edit" (click)="goHabitMore()"></div>
```
Comment (Edw1x) [inline src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.html:15]: can you make a more appropriate name for the function? If it edits something maybe consider a name like  'goEditHabit'

Comment (idzhersh) [reply src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.html:15]: changed

Comment (idzhersh) [reply src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.html]: ### changed

**Review by paslavskyi9roman — APPROVED** [review]

**Review by Edw1x — APPROVED** [review]

File: src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.html [hunk 2]
```diff
@@ -12,6 +12,7 @@
         <img class="first-photo" [src]="firstFriend" alt="First friend photo">
         <img class="second-photo" [src]="secondFriend" alt="Second friend photo">
       </div>
+      <div class="edit" (click)="goEditHabit()"></div>
```
Comment (JimHolman1199) [inline src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.html:15]: what is goEditHabit ?

File: src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.ts [hunk 3]
```diff
@@ -26,21 +26,21 @@ export class OneHabitComponent implements OnInit {
...
-      this.habitMark = 'mark';
+      this.habitMark = 'done';
     },
     undone: () => {
       this.daysCounter = this.habit.workingDays
         ? this.habit.workingDays
         : this.habit.duration;
       this.showPhoto = true;
-      this.habitMark = 'empty';
+      this.habitMark = 'undone';
```
Comment (JimHolman1199) [inline src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.ts:43]: create enum in this file for habitMarkStatuses

Comment (idzhersh) [reply src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.ts]: done

File: src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.html (hunk 2 above)
Comment (idzhersh) [reply src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.html]: changed

Comment (idzhersh) [reply src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.html]: This is function to navigate to habit profile

**Review by yuriiLisunov — APPROVED** [review]

**Review by KirillAbramo — APPROVED** [review]

**Review by AlekOK — APPROVED** [review]

**Review by JimHolman1199 — APPROVED** [review]

**Review by mr-pavlenko — APPROVED** [review]

**Pull request merged**